scripts/sign_windows_executable.js

```diff
@@ -0,0 +1,169 @@
+// Copyright 2022 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const minimist = require('minimist');
```

Comment:
Why do we have to use CommonJS here? is it because of jsign?

Reply:
If you wanted to get the drop on supporting node actions, maybe we create a `src/electron/sign.action.mjs` and update `scripts/run_action.sh` to look for `*.action.mjs` files?

Reply:
No, it is because electron-builder only supports CommonJS. I explained it in the comment above `exports.default = electronBuilderEntryPoint;`.

Reply:
mm - perhaps that's a case for calling electron-builder in bash for now, so we can write as much in ESModules as possible, but I'll leave it up to @fortuna 

if we add support for calling mjs as actions later we could add a precompile step, so this approach would be future-facing towards that

Reply:
I updated the file structure, now only `electron_builder_signing_plugin.cjs` will use CommonJS.